### PR TITLE
Add the email_spam_detection directory in NLP/projects

### PR DESCRIPTION
**Pull request by ikcod**
Added the email_spam_detection project in NLP/projects

Closes: #86 

- **Title**  :  Email_Spam_Detection
- **Your Name:**  Insha Khan
- **Idenitfy yourself:** KwoC Contributor

### Describe the add-ons or changes you've made 📃

- I have added the email_spam_detection project in the NLP/projects directory
- This project contatins the jupyter file , dataset used in the project and a read me file that explains the project

### Checklist: ☑️
- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/AI-Code/blob/main/README.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/AI-Code/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly wherever it was hard to understand.
- [x] My changes generate no new warnings.

### CODE REVIEW

File: docs/NLP/projects/Email_Spam_Detection/email.ipynb
Comment (Avdhesh-Varshney) [inline docs/NLP/projects/Email_Spam_Detection/email.ipynb:1]: Upload it either on drive or kaggle.
Just attach the link of the notebook.

File: docs/NLP/projects/Email_Spam_Detection/mail_data.csv
Comment (Avdhesh-Varshney) [inline docs/NLP/projects/Email_Spam_Detection/mail_data.csv:1]: Dataset not required.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
You can take refrence from here and also attach a video how your project page looks like.

`https://avdhesh-varshney.github.io/AI-Code/NLP/projects/twitter_sentiment_analysis/`

**Comment by ikcod** [comment]
Completed the required changes.

File: docs/NLP/projects/Email_Spam_Detection/README.md [hunk 1]
```diff
@@ -0,0 +1,213 @@
+
+# Email Spam Detection
+
+### AIM 
+To develop a machine learning-based system that classifies email content as spam or ham (not spam).
+
+### DATASET LINK 
+[https://www.kaggle.com/datasets/ashfakyeafi/spam-email-classification](link)
```
Comment (Avdhesh-Varshney) [inline docs/NLP/projects/Email_Spam_Detection/README.md:8]: link?

File: docs/NLP/projects/Email_Spam_Detection/README.md [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+
+
+### NOTEBOOK LINK 
+[https://www.kaggle.com/code/inshak9/email-spam-detection](link)
```
Comment (Avdhesh-Varshney) [inline docs/NLP/projects/Email_Spam_Detection/README.md:12]: link

File: docs/NLP/projects/Email_Spam_Detection/README.md [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+### FEATURES PLANNED BUT NOT IMPLEMENTED
+
+=== "Feature 1"
+
+    - Integration of deep learning models (LSTM) for improved accuracy.
+
+---
+
+### **DEVELOPER**
+***Your Name***
```
Comment (Avdhesh-Varshney) [inline docs/NLP/projects/Email_Spam_Detection/README.md:202]: .

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
Attach a video

**Comment by ikcod** [comment]
Video of project page after adding the links to dataset and kaggle notebook

https://github.com/user-attachments/assets/7619d706-48a9-4725-9f83-4207cbc51291

**Review by Avdhesh-Varshney — APPROVED** [review]
It looks good. But there are some issues like,
- Committed directly in `main` branch.
- I required a video of the project documentation page not project github page.

You should join the discord server/ github discussion for faster communication.

**Comment by Avdhesh-Varshney** [comment]
https://avdhesh-varshney.github.io/AI-Code/NLP/projects/Email_Spam_Detection/

Review this project page on your own.

**Pull request merged**